server/sessionManagementRoutes.js

```diff
@@ -103,8 +97,11 @@ const configureRoutes = ({ app, healthApi, oauthApi, tokenRefresher, mailTo }) =
   app.get('/login', loginMiddleware, loginIndex)
   app.post('/login', login)
   app.get('/auth/logout', logout)
+  app.get('/logout', (req, res) => {
+    res.redirect('/auth/logout')
```

Comment:
not sure why we have paths for logout.  This app is all about `/logout` but the common header takes the user to `/auth/logout` instead